shopinvader_lead/models/lead.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Akretion (http://www.akretion.com).
+# @author Sébastien BEAU <sebastien.beau@akretion.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class ShopinvaderLead(models.Model):
+    _inherit = 'crm.lead'
```

Comment:
@sebastienbeau it's not a shopinvader.lead.... The naming is wrong (file an model)